src/systems/systems/src/pressurization/mod.rs

```diff
@@ -106,6 +224,114 @@ impl Default for Pressurization {
     }
 }
 
+pub struct PressurizationOverheadPanel {
+    mode_sel: AutoManFaultPushButton,
+    man_vs_ctl_switch: SpringLoadedSwitch,
+    ldg_elev_knob: ValueKnob,
+    ditching: NormalOnPushButton,
```

Comment:
**note**: The same Law of Demeter violation mentioned earlier with regards to `ldg_elev_knob` also exists for other fields in this type. For example:

```rust
press_overhead.mode_sel.is_man()
```
could be changed to something like:
```rust
press_overhead.mode_is_manual()
```

Reply:
Corrected, this was my mistake as I already had a function to do this: `is_in_man_mode()`